doc/source/api.rst

```diff
@@ -1,9 +0,0 @@
-`ixmp` API Documentation
-------------------------
-
-.. include:: api/ixmp.rst
-
-`message_ix` API Documentation
-------------------------------
-
-.. include:: api/message_ix.rst
```

Comment:
Also made these changes in #138, but I can rebase that other PR after this one is merged.